src/PluginClient.php

```diff
@@ -79,7 +78,7 @@ public function sendRequest(RequestInterface $request)
         $pluginChain = $this->createPluginChain($this->plugins, function (RequestInterface $request) {
             try {
                 return new FulfilledPromise($this->client->sendRequest($request));
-            } catch (HttplugException $exception) {
+            } catch (\Exception $exception) {
```

Comment:
If one have written an function HttplugException (just like we did in HistoryPlugin) it will fail with this update. But as @GrahamCampbell says, the rejected promise class always said it would be an \Exception. 

So: This is a bugfix, others may also suffer from this bug. But this is no BC break.